interfaces/udev/backend_test.go

```diff
@@ -322,6 +323,22 @@ func (s *backendSuite) TestCombineSnippetsWithActualSnippets(c *C) {
 	}
 }
 
+func (s *backendSuite) TestControlsDeviceCgroup(c *C) {
+	// NOTE: Hand out a permanent snippet so that .rules file is generated.
+	s.Iface.UDevPermanentSlotCallback = func(spec *udev.Specification, slot *snap.SlotInfo) error {
+		spec.SetControlsDeviceCgroup()
+		spec.AddSnippet("dummy")
+		return nil
+	}
+	for _, opts := range testedConfinementOpts {
+		snapInfo := s.InstallSnap(c, opts, "", ifacetest.SambaYamlV1, 0)
+		fname := filepath.Join(dirs.SnapUdevRulesDir, "70-snap.samba.rules")
+		_, err := os.Stat(fname)
+		c.Assert(err, ErrorMatches, fmt.Sprintf("stat.*%s.*no such file or directory", fname))
```

Comment:
I think we have a testutil.FileAbsent checker nowadays (is not very used yet) for this, or also osutil.FileExists

Reply:
Great, thanks for the tip! I used `testutil.FileAbsent` here instead as that's a more direct and clear test